src/main/java/com/pdffiller/client/utils/MultipartHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.pdffiller.client.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.security.SecureRandom;
+import java.util.HashMap;
+
+public class MultipartHelper {
+  private static final String CRLF = "\r\n";
+  private File file;
+  private String boundary;
+  private byte[] message;
+
+  public File getFile() {
+    return file;
+  }
+
+  public void setFile(File file) {
+    this.file = file;
+  }
+
+  public byte[] getMessage() {
+    return message;
+  }
+
+  public MultipartHelper(String file) {
+    this.file = new File(file);
+    boundary = generateBoundary();
+    message = formRequest();
+  }
+
+  private String generateBoundary() {
+    SecureRandom random = new SecureRandom();
+    return new BigInteger(20, random).toString();
+  }
+
+  private String getMime() {
+    String mime = null;
+    try {
+      mime = Files.probeContentType(file.toPath());
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    return mime;
+  }
+
+  public byte[] getFileContent() {
+    byte[] content = new byte[0];
+    try {
+      content = Files.readAllBytes(file.toPath());
+    } catch (IOException e) {
```

Comment:
Please don't just throw unterminated exceptions:

Do this:
 } catch (IOException e) {
            e.printStackTrace();
            throw new PdfFillerAPIException(e.getMessage());
        }